winui/Scheduler/Appointment-Editing.md

```diff
@@ -50,12 +72,6 @@ The [AppointmentEditorOpeningEventArgs](https://help.syncfusion.com/cr/winui/Syn
 
 `Cancel`: To avoid the default appointment editor showing, by enabling this property.
```

Comment:
`Cancel:` To avoid the default appointment editor from showing by enabling this property.